run-tests.php

```diff
@@ -2959,8 +2959,34 @@ function generate_array_diff(array $ar1, array $ar2, bool $is_reg, array $w): ar
     return $diff;
 }
 
+function generate_diff_external(string $wanted, string $output): string
+{
+    global $environment;
+
+    $diff_cmd = $environment['DIFF_CMD'];
+
+    /* Write two temp files */
+    $wanted_file = $environment['TEMP'] . DIRECTORY_SEPARATOR . 'wanted.txt';
```

Comment:
There should already be files written to disk for this ( `.exp` and `.out`)

Reply:
Oh yes, of course!